README.rst

```diff
@@ -1,5 +1,66 @@
 Kebechet
 --------
 
-I'm Kebechet, goddess of freshness. I will keep your dependencies fresh and up-to-date.
+  I'm Kebechet, goddess of freshness. I will keep your dependencies fresh and up-to-date.
 
+Kebechet is a bot that automates dependency management for your projects. Currently, it supports managing and updating Python projects based on pipenv files (``Pipfile`` and ``Pipfile.lock``) or ``requirements.txt``/``requirements.in`` files.
```

Comment:
Kebechet is a SourceOps bot that automates updating dependencies of your project. Currently...